### PR TITLE
Stop docker image build during PR and cleaned the docker CI file

### PR DESCRIPTION
**Pull request by calvinp0**

### CODE REVIEW

**Review by alongd — COMMENTED** [review]
LGTM, please rebase

File: .github/workflows/docker_build.yml [hunk 1]
```diff
@@ -1,44 +1,32 @@
...
-        uses: actions/checkout@v2
-      - name: Login to Docker Hub
-        uses: docker/login-action@v1
-        with:
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1
-      
-      - name: Debug
```
Comment (Lilachn91) [inline .github/workflows/docker_build.yml:28]: Why did you remove the debug step?

Comment (calvinp0) [reply .github/workflows/docker_build.yml:28]: Because it isn't required for the main branch - I just needed it during the development
